composer.json

```diff
@@ -31,7 +31,8 @@
         "doctrine/dbal": "~2.3",
         "jdorn/sql-formatter": "~1.1",
         "symfony/doctrine-bridge": "~2.2|~3.0",
-        "doctrine/doctrine-cache-bundle": "~1.0"
+        "doctrine/doctrine-cache-bundle": "~1.0",
+        "symfony/web-profiler-bundle": "~2.7|~3.0"
```

Comment:
Yes, which is why I'm not sure whether we need to add this. On the other hand, we should make sure the `profiler_dump` twig function exists. Dropping Symfony 2.3 would probably implicitly ensure it is, but I'd rather have it explicit.